weblate/auth/models.py

```diff
@@ -295,6 +295,10 @@ def all_admins(self, project):
         """All admins in a project."""
         return self.having_perm("project.edit", project)
 
+    def all_reviewers(self, project):
```

Comment:
Let's have a type annotation on the new code:

```suggestion
    def all_reviewers(self, project : Project) -> UserQuerySet:
```